CHANGELOG.md

```diff
@@ -56,6 +58,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   ([#1333](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1333))
 - Use resp.text instead of resp.body for Falcon 3 to avoid a deprecation warning.
   ([#1412](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1412))
+- `opentelemetry-instrumentation-pymysql` Fix dbapi connection instrument wrapper has no _sock member.
+  ([#1424](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1424))
+- `opentelemetry-instrumentation-dbapi` Fix the check for the connection already being instrumented in instrument_connection().
+  ([#1424](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1424))
```

Comment:
Please move CHANGELOG entries to Unreleased section.

Reply:
Done!